Makefile

```diff
@@ -453,19 +454,13 @@ opt-core: runtimeopt
 
 .PHONY: opt
 opt:
-ifeq "$(UNIX_OR_WIN32)" "unix"
 	$(MAKE) runtimeopt
 	$(MAKE) ocamlopt
 	$(MAKE) libraryopt
 	$(MAKE) otherlibrariesopt ocamltoolsopt
-else
-	$(MAKE) opt-core
-	$(MAKE) otherlibrariesopt ocamltoolsopt
-endif
```

Comment:
Wouldn't it be better to have adopted the Windows version here?

```make
opt: opt-core
	$(MAKE) otherlibrariesopt ocamltoolsopt
```

and make the relationship between `opt` and `opt-core` explicit?

Reply:
If you look at the file's history you'll see a lot of back-and-forth between dependencies and recursive calls to `$(MAKE)`, all of them committed to fix parallel make. In my eye, all these versions are equivalent. I probably chose this one with the idea that opt-core would disappear eventually.

Reply:
Oh, and I don't like having a phony target as a dependency because then it's not clear that it will be built every time. Better to call it explicitly IMO.